packages/client/src/__tests__/server-side/call-types.test.ts

```diff
@@ -92,22 +89,22 @@ describe('call types CRUD API', () => {
     });
   });
 
-  it('update', async () => {
```

Comment:
I'd rather use `it.skip` instead of commenting out the whole test.

Reply:
Agh okay, let me fix this